mod/bootstrap/set/all/bootstrap/icon.rb

```diff
@@ -32,7 +32,9 @@
     reset: :restart_alt,
     more: :more_horiz,
     menu: :more_vert,
-    bar_menu: :more_horiz
+    bar_menu: :more_horiz,
+    eye_open: :visibility,
+    eye_slash: :visibility_off
```

Comment:
I would think it would be more intuitive to use something like `show_password` and `hide_password` for our keys here. More importantly, we want to make sure that we use these keys in the code. (You have used the material name rather than the decko name in the code).

Also, we generally want to try to make it so that we have all the icon keys mapped in font awesome, too, so that someone can switch over to font awesome without things breaking. Can you add these new ones in font awesome as well?